src/tests/Microsoft.PowerFx.Core.Tests/ExpressionTestCases/DecimalNonMathFuncs_ColumnNamesAsIdentifiers.txt

```diff
@@ -0,0 +1,15 @@
+#SETUP: disable:NumberIsFloat
+#SETUP: SupportColumnNamesAsIdentifiers
+
+>> AddColumns(["1","2","3"],dec,Decimal(Value)+0.00000000000000000000001)
+Table({Value:"1",dec:1.00000000000000000000001},{Value:"2",dec:2.00000000000000000000001},{Value:"3",dec:3.00000000000000000000001})
+
+>> AddColumns(["1","2","3"],dec,Value+0.00000000000000000000001)
+Table({Value:"1",dec:1.00000000000000000000001},{Value:"2",dec:2.00000000000000000000001},{Value:"3",dec:3.00000000000000000000001})
+
+>> AddColumns(["1","2","3"],dec,Float(Value)+0.00000000000000000000001)
+Table({Value:"1",dec:1},{Value:"2",dec:2},{Value:"3",dec:3})
```

Comment:
[minor] Since the type of the added column is not decimal (it is float), can you name it accordingly, so it doesn't confuse who reads this test?

Reply:
Good call.  I moved these tests to AddColumns files.  We don't need the C# decimal variants for this, the DV decimal will suffice.